### PR TITLE
Update adapter-settings, add ZB-OTA info

### PR DESCRIPTION
**Pull request by Tarik2142**
Update adapter-settings https://github.com/Koenkk/zigbee-herdsman/pull/704
Added information about the possibility of OTA for Zigbee coordinators

### CODE REVIEW

File: docs/guide/configuration/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The file have to be located in the `data` directory within your installation. Th
 permit_join: true
 mqtt:
   server: mqtt://localhost:1883
+# Could be either USB port (/dev/ttyUSB0) or Ethernet port for Ethernet Zigbee adapters (tcp://192.168.1.1:6638).
```
[the remark this answers is not in the document]
Comment (Tarik2142) [reply docs/guide/configuration/README.md]: +

File: docs/guide/configuration/adapter-settings.md [hunk 2]
```diff
@@ -4,15 +4,33 @@ sidebarDepth: 1
...
 
+## Configuration of the Zigbee-Network(Ethernet/Wi-Fi) Adapter. 
+Find IP address of your  Zigbee-Network Adapter through your router/switch web-interface.
 ```yaml
 # Required: serial settings
 serial:
   # Required: location of the adapter (e.g. CC2531).
-  # To autodetect the port, set 'port: null'.
-  port: /dev/ttyACM0
+  # Ethernet adapters - use format "port: tcp://IP_adress:port".
```
[the remark this answers is not in the document]
Comment (Koenkk) [reply docs/guide/configuration/adapter-settings.md:32]: MDNS Zeroconf discovery. can be left separate, so just Zigbee-Network(Ethernet/Wi-Fi) Adapter.  and Zigbee USB-Adapter.

[the remark this answers is not in the document]
Comment (Tarik2142) [reply docs/guide/configuration/adapter-settings.md:32]: ок

[the remark this answers is not in the document]
Comment (Tarik2142) [reply docs/guide/configuration/adapter-settings.md]: @Koenkk added, changes. Is everything ok?

**Pull request merged**